uprotocol/uattributes.proto

```diff
@@ -71,9 +72,8 @@ message UAttributes {
 
     // Communication error attribute populated by uP-L2 dispatchers only when an error 
     // has occurred in the delivery of RPC request or response events. 
-    // The contents of this attribute, if present, is the unsigned integer representation of
-    // UCode. If the attribute is not present, there is no communication error
-    optional uint32 commstatus = 8;
+    // If the attribute is not present, there is no communication error
+    optional UCode commstatus = 8;
```

Comment:
FMPOV this should be changed according to the outcome of https://github.com/eclipse-uprotocol/up-core-api/issues/77

Reply:
@sophokles73 are you saying the documentation would need to change but not the field name and purpose right? We might add additional UCodes but it shouldn't change the signature

Reply:
Well, *communication error* is quite a generic term. IMHO it could be used to indicate both a problem at the infrastructure or the application level. This is what I suggested in #77. The only thing that we need to do is to define the semantics of *commstatus* accordingly in the spec.